### PR TITLE
Sf parameters

### PR DESCRIPTION
**Pull request by Avilir**
updating the benchmark parameters names to align with the upstream benchmark names, so we can collect test results.

### CODE REVIEW

File: tests/e2e/performance/io_workload/test_small_file_workload.py [hunk 1]
```diff
@@ -249,6 +250,7 @@ def combine_results(self, results, clear):
             res.update(self.aggregate_threads_results(res))
         else:
             res = self.aggregate_threads_results(res)
+        log.debug(f"The combines results are : {json.dumps(res, indent=2)}")
```
Comment (ypersky1980) [inline tests/e2e/performance/io_workload/test_small_file_workload.py:253]: small spelling fix: 

"The combined results" instead of "The combines results"

**Review by ypersky1980 — APPROVED** [review]

**Review by pintojoy — APPROVED** [review]
LGTM

File: tests/e2e/performance/io_workload/test_small_file_workload.py [hunk 2]
```diff
@@ -468,12 +479,21 @@ def generate_kibana_link(self, index, columns):
...
-        kibana_id = self.get_kibana_indexid(
-            self.crd_data["spec"]["elasticsearch"]["host"],
-            index,
-        )
+        log.info(json.dumps(self.crd_data.get("spec").get("elasticsearch"), indent=2))
+        host = self.crd_data.get("spec").get("elasticsearch").get("url")
+        try:
+            host = host.split(":")[1].replace("//", "")
+        except Exception:
+            log.error("No ES configuretion")
```
Comment (ramkiperiy) [inline tests/e2e/performance/io_workload/test_small_file_workload.py:487]: minor spell check

**Review by keesturam — APPROVED** [review]
LGTM

**Pull request merged**